.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,30 @@
+name: Ansible Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        lfs: false # not needed currently
```

Comment:
Would the comment still make sense if it were reworded as "not needed for this job"?  If so, that might be clearer, and then ditto for line 26.

Reply:
Done.  `lfs: false` is the default, but I wanted to make it explicit that this wasn't an oversight here.